server/resources/resources.test.js

```diff
@@ -242,7 +242,7 @@ describe("/api/resources", () => {
 
 			await anonAgent
 				.get("/api/resources")
-				.query({ draft: true })
+				.query({ draft: true, account: false })
```

Comment:
❓ Isn't this the implicit default?

❌ There are no tests for the new behaviour